styles/styles.css

```diff
@@ -435,6 +554,154 @@ body {
     width: calc(var(--col-width) * 12);
   }
 
+  .top__wrapper {
+    padding: 30px 35px;
+  }
+
+  .top_header {
+    margin-bottom: 52px;
+  }
+
+  .top__logo {
+    height: 35px;
+    width: 180px;
+  }
+
+  .top__menu {
+    flex-basis: 120px;
+  }
+
+  .top__list {
+    padding-right: 32px;
+  }
+
+  .top__icon-night {
+    width: 25px;
+    height: 25px;
+  }
+
+  .top__icon-account {
+    width: 35px;
+    height: 35px;
+  }
+
+  .top__list-mobile {
+    background-image: url(../image/menu.png);
+    width: 30px;
+    height: 20px;
+    align-self: flex-start;
+    order: 1;
+  }
+
+  .top__item {
+    display: none;
+  }
+
+  .top__nav-item {
+    order: 2;
+  }
+
+  .top__nav-icon {
+    order: 1;
+  }
+
+  .top__title {
+    padding: 120px 0 31px;
+    max-width: 380px;
+    font-size: 30px;
+  }
+
+  .top__text {
+    font-size: 35px;
+  }
+
+  .top__form {
+    display: flex;
+    flex-direction: column;
+    margin-block-start: 0px;
+    background-color: inherit;
+    padding-bottom: 50px;
+  }
+
+  .top__input-name {
+    height: 48px;
+    border-radius: 0;
+    border: 4px solid var(--font-bright);
```

Comment:
Your border size is right for top and bottom.  but for left and right it must be 8px. So it's touched all you left and right borders on mobile form